src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -383,7 +384,8 @@ class BinaryCompactObject : public DomainCreator<3> {
     static double lower_bound() { return 0.; }
     static double upper_bound() { return 1.; }
     // Suggest spherical frustums to encourage upgrading, but keep supporting
-    // cubical frustums until spherical frustums are sufficiently battle-tested
+    // cubical frustums until spherical frustums are sufficiently
+    // battle-tested
```

Comment:
drop this formatting change (is your editor line width set to 80 chars correctly?)